### ISSUE TITLE
Search results info button (/status/*hash*) 404

### ISSUE THREAD

**Issue by ShoGinn** [nosmokingbandit/watcher#86]
Using NZBHydra, the search results have an "i" link that is a 404 page.

Not sure what the original intent was.

With a reverse proxy enabled here is a link example.

http://*server*/watcher/status/585d128b9cc1aff167431a7e61fec174cdea2b3c

**Comment by soundjester** [comment]
I use NZBHydra as well, and I have an idea of why this is happening.  In the XML, Hydra passes along a link to the "detail" information for the NZB in the `<comments>` tag:

```
<item>
    <title>A.Movie.Title.of.Some.Kind.1080p-FraMeSToR</title>
    <guid isPermaLink="false">d9095e41a3ad02c679a2c5cb550d9f5e410eb4c5</guid>
    <link>http://192.168.0.10:5075/getnzb?...GUID Link to download the NZB File</link>
    <comments>https://my.nzbindexer.com/details/72c02819e0ca10e0995d15e4213d44ac</comments>
    <pubDate>Sat, 31 Dec 2016 22:13:46 -0600</pubDate>
    <category>Movies HD</category>
    <description>A.Movie.Title.of.Some.Kind.1080p-FraMeSToR - MyNZBIndexer</description>
    <newznab:attr />
        ...  long list of newznab attributes ...
    <newznab:attr >
</item>
```

This tag isn't picked up by the corresponding [core/newznab.py](https://github.com/nosmokingbandit/watcher/blob/c3ae2e65896ccc303fab73063960555095f5657f/core/newznab.py#L91) script, and the changing it to incorporate this information would adversely affect link writing (and possibly other things) downstream.

We'll have to see if there is an elegant solution to this to be found.

**Comment by nosmokingbandit** [comment]
NZBHyrda doesn't follow typical NwezNabb formatting, so there are many exceptions that must be put in place to make it work. Things end up getting really hacky if I'm not careful.

**Comment by ShoGinn** [comment]
I tried to see if that would work; I still get a 404.

**Comment by soundjester** [comment]
Same (404).  I don't think I would stretch very far at this stage to accommodate NZBHydra for this feature.  If, at a later date, you decide to accommodate custom search providers, we can revisit this.

**Comment by nosmokingbandit** [comment]
So many people use NZBHydra, so I'd like this to work. I'll just have to set it up and run a few tests myself.

**Comment by ShoGinn** [comment]
maybe @theotherp would come and try and help ;)

**Comment by nosmokingbandit** [comment]
Maybe someone can help me set up nzbhydra.

This is all I get when I search. I just added 6box and newztown to test.

```
2017-01-02 22:09:14,782 - INFO - web - Thread-12 - API search request: SearchRequest [ type "movie", identifier_key "imdbid" , identifier_value "0947798", forbiddenWords "[]", requiredWords "[]", category "Movies", offset "0", loadAll "False", limit "100" ]
2017-01-02 22:09:14,782 - INFO - search - Thread-12 - Starting new search: SearchRequest [ type "movie", identifier_key "imdbid" , identifier_value "0947798", forbiddenWords "[]", requiredWords "[]", category "Movies", offset "0", loadAll "False", limit "100" ]
2017-01-02 22:09:14,790 - WARNING - search - Thread-12 - No indexeres were selected for this search:
Does not support ID based searches: NewzTown
Disabled for this category Movies: Womble, 6box
Disabled for API searches: Binsearch, NZBClub, NZBIndex
2017-01-02 22:09:14,826 - INFO - search - Thread-12 - All indexers exhausted
2017-01-02 22:09:14,828 - INFO - search - Thread-12 - Returning 0 results
2017-01-02 22:09:14,832 - INFO - web - Thread-12 - API search request completed in 50 ms
```

**Comment by soundjester** [comment]
In the Indexers config, you need to make sure that your provider is enabled for both internal and API searchers (the drop-down is towards the bottom).  You also may need to to the "Check Capabilities" toggle at the very bottom for each indexer.

The `Disabled for this category Movies: Womble, 6box` line gives it away.

**Comment by nosmokingbandit** [comment]
@soundjester 
I finally figured it out. NZBH, by default, wasn't accepting imdb searches for my added indexers. I had to edit the indexer settings to allow searches.

Commit https://github.com/nosmokingbandit/watcher/commit/ddc147b3cd6e440488c8da38d12f3291de0de39e should fix this.

It won't change any existing search results, but new ones will have the correct info link. If you remove then re-add a movie it will purge all existing search results and the new results will have fixed links.
